addon.xml

```diff
@@ -3,6 +3,7 @@
     <requires>
         <import addon="xbmc.addon" version="16.0.0" />
         <import addon="repository.elementum" optional="true" />
+        <import addon="script.module.kodi-six" />
```

Comment:
kodi-six was used locally because it has a fix for Kodi16 - https://github.com/elgatito/plugin.video.elementum/commit/953e3bacbc40a7ed26db5807f9cfa9c2b3f54b3c
Does repo version has it?

Reply:
Не в курсе, надо будет проверить

Reply:
@elgatito Только что тестировал на kodi 16, все работает.

Reply:
@Lontron @vlmaksime Баг с Коди 16 проявляется не всегда, а при достижении этого куска кода. 
Я у себя проверить не могу. Нужно смотреть какой код в этом модуле в Коди 16. Если фикса нету то вырезать нельзя из дополнения kodi-six.